test/e2e/page-objects/pages/swap/swap-page.ts

```diff
@@ -84,6 +153,59 @@ class SwapPage {
       tag: 'button',
     });
   }
+
+  async checkNoQuotesAvailable(): Promise<void> {
+    await this.driver.waitForSelector({
+      text: `This trade route isn't available right now. Try changing the amount, network, or token and we'll find the best option`,
+      tag: 'p',
+    });
+  }
+
+  async createSolanaSwap(options: SwapSolanaOptions) {
+    await this.driver.clickElement(this.bridgeSourceButton, 3);
+    await this.driver.delay(2000);
+
+    await this.driver.clickElement({
+      text: options.swapFrom,
+      css: '[data-testid="multichain-token-list-button"] p',
+    });
+
+    await this.driver.clickElement(this.bridgeDestinationButton, 3);
+
+    await this.driver.clickElement({
+      text: options.swapTo,
+      css: '[data-testid="multichain-token-list-button"] p',
+    });
+
+    await this.driver.waitForSelector(this.fromAmount);
+    await this.driver.fill(this.fromAmount, options.amount.toString());
+  }
+
+  async reviewSolanaQuote(options: SwapSolanaReviewOptions) {
+    await this.driver.waitForSelector(this.submitSwapButton);
+    const fromAmount = await this.driver.findElement(
+      '[data-testid="from-amount"]',
+    );
+    const fromAmountText = await fromAmount.getAttribute('value');
+    assert.equal(fromAmountText, options.swapFromAmount);
+    const toAmount = await this.driver.findElement('[data-testid="to-amount"]');
+    const toAmountText = await toAmount.getAttribute('value');
+    assert.equal(toAmountText, options.swapToAmount);
+    await this.driver.waitForSelector({
+      text: `1 ${options.swapFrom} = ${options.swapToAmount} ${options.swapTo}`,
+      tag: 'p',
+    });
+    await this.driver.waitForSelector({
+      text: `Rate includes 0.875% fee`,
+      tag: 'p',
+    });
+    await this.driver.waitForSelector({
+      text: `More quotes`,
+      tag: 'button',
+    });
```

Comment:
could we define a selector above, and then use the defined selector, instead of hardcoding it here?

Reply:
updated